src/views/AccountSettings.vue

```diff
@@ -51,6 +53,7 @@ export default {
 		EditorSettings,
 		Navigation,
 		SignatureSettings,
+		AliasSettings,
```

Comment:
you'll get bonus points if you arrange the props alphabetically :wink:

Reply:
Done